recipes/nextalign/meta.yaml

```diff
@@ -49,3 +50,6 @@ extra:
   skip-lints:
     # repackaged binary
     - should_be_noarch_generic
+  identifiers:
+    - doi:10.21105/joss.03773
+    - doi:10.21105/joss.03773
```

Comment:
```suggestion
```

Reply:
seems there is a doubleup of the DOI

Reply:
ah woops, not sure how that happened. Thanks for the catch!